File: web3rpc/rpc-specs/code-samples/python/klay/account/getCode.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
blockTag = "0x2"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_code(address, blockTag)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/account/getCode.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const account = "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b";
  const blockNumberOrHash = "0x2";

  provider.klay
    .getCode(account, blockNumberOrHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
